.circleci/config.yml

```diff
@@ -0,0 +1,183 @@
+version: 2
+jobs:
+  py2:
+    machine: true
+    steps:
+      - run: sudo add-apt-repository -y ppa:terry.guo/gcc-arm-embedded
+      - run: sudo apt-get -y update
+      - run: sudo apt-get -y install gcc-arm-none-eabi=4.9.3.2015q3-1trusty1
```

Comment:
Can we update this to GCC 6.x? Pretty sure Mbed OS doesn't [recommend using 4.9.x anymore](https://os.mbed.com/docs/v5.8/tools/index.html).

Reply:
Woah. You're right, but I'd prefer to change only what's needed for the topic at hand.

Definitely going to need to submit another PR to get that updated.

Reply:
Updating GCC 6.x would only work for newer Mbed OS, but older cases will fail (like Morpheus). And the Morpheus routine is used for comprehensive multi-level dependencies handling, including removal of libraries etc.

Reply:
That make sense, but we should be testing with the recommended toolchain for each Mbed OS release. So if that means installing two different versions of GCC then we may need to do that.